deploy/internal/deployment-endpoint.yaml

```diff
@@ -23,6 +23,16 @@ spec:
         noobaa.io/configmap-hash: ""
     spec:
       serviceAccountName: noobaa-endpoint
+      topologySpreadConstraints:
+      - maxSkew: 1
+        topologyKey: kubernetes.io/hostname
+        whenUnsatisfiable: ScheduleAnyway
+        # nodeTaintsPolicy available from kubernetes 1.26 or latest, 
+        # without this master node also will use for calculating pod topology spread skew
+        nodeTaintsPolicy: Honor
```

Comment:
Could not find `nodeTaintsPolicy` in [openshift documentation](https://docs.openshift.com/container-platform/4.13/nodes/scheduling/nodes-scheduler-pod-topology-spread-constraints.html) , only in the [kubernetes documentation](https://kubernetes.io/docs/concepts/scheduling-eviction/topology-spread-constraints/) 

Does Openshift support this? 
if yes, in what version, if not what will happen?

Reply:
this is supported since [kuberentes 1.19](https://kubernetes.io/blog/2023/04/17/fine-grained-pod-topology-spread-features-beta/)